src/handlers/push/index.ts

```diff
@@ -26,6 +26,6 @@ export const runOnPush = async () => {
   // check for modified or added files and check for specified file
   if (payload.head_commit.modified.includes(BASE_RATE_FILE) || payload.head_commit.added.includes(BASE_RATE_FILE)) {
     // update base rate
-    updateBaseRate(context, payload, BASE_RATE_FILE);
+    await updateBaseRate(context, payload, BASE_RATE_FILE);
```

Comment:
I'm pretty sure I fixed this in the original PR. How was it removed?

Reply:
when I catch it up, it was not there. so I think I added it again.

Reply:
I made a mistake and realized that it is a different function with a similar name that I fixed in https://github.com/ubiquity/ubiquibot/pull/352#discussion_r1216587731

Reply:
Yeah that was the only part but I can go through it.

Reply:
I don't think we should fix `await` problems in this PR. I suggest to do that out of this PR.